tests/Benchmarks/Microsoft.Experimental.Collections/DictionarySlimFasta.cs

```diff
@@ -0,0 +1,279 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Collections.Concurrent;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Experimental.Collections.Benchmarks
```

Comment:
Hmm, my bad. I just realized it is not clear what the license is for this file (if any). I do not see mention on the site, and they no longer have a public repo. I think unless and until we can persuade the maintainer to make the license clearer, we cannot include it in the repo even as test code. 

What about if we check the 3 inputs in zipped up. (Looks like the inputs are incredibly compressible, even with the 250MB one the zip is only 420KB.) Then decompress in code (instead of running Fasta)?

Reply:
The gitlab repo is here: https://salsa.debian.org/benchmarksgame-team/benchmarksgame. Is the licence ok?

Reply:
Let me ask experts

Reply:
At first I couldn't find the sources in there but looks like they're in a [zip file](https://salsa.debian.org/benchmarksgame-team/benchmarksgame/blob/master/public/download/benchmarksgame-script.zip). (In fact I think it was me that requested that Isaac add them.)

I asked and it's OK for us to include the file, so long as we give attribution in a third party notices file, which should go in the same folder, like this one:
https://github.com/dotnet/coreclr/blob/master/tests/src/JIT/Performance/CodeQuality/Burgers/THIRD-PARTY-NOTICES . The content would be https://salsa.debian.org/benchmarksgame-team/benchmarksgame/blob/master/LICENSE.md

Reply:
Same content added at the top
```
.NET Core uses third-party libraries or other resources that may be
distributed under licenses different than the .NET Core software.

In the event that we accidentally failed to list a required notice, please
bring it to our attention. Post an issue or email us:

           dotnet@microsoft.com

The attached notices are provided for information only.
```